tensorboard/webapp/metrics/views/card_renderer/scalar_card_types.ts

```diff
@@ -99,6 +99,11 @@ export interface ColumnHeader {
   enabled: boolean;
 }
 
+export enum FobState {
```

Comment:
"FobState" seems like the wrong name. Fob is a tool related to step selection and linked time but not really a clear way to refer to either feature.

Maybe DataTableMode or StepSelectionMode?

Reply:
I like DataTableMode.